_slides/08-kalyan0816.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Welcome Riddlocat!! Kalyan became Riddlocat for Github Training. Hahaha
```

Comment:
Nice caption.

Can we consider shortening this caption?
[Docs](reasoning.com)